<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapFormClass</class>
 <widget class="QWidget" name="MapFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget_map">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>cod</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_map">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_map_destination">
       <property name="text">
        <string>destination:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_findpath_x"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_findpath_y"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_findpath_start">
       <property name="text">
        <string>findpath</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_findpath_stop">
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
